bogo/Cargo.toml

```diff
@@ -6,6 +6,6 @@ edition = "2021"
 [dependencies]
 base64 = "0.22"
 env_logger = "0.10" # 0.11 requires 1.71 MSRV even as a dev-dep (due to manifest features)
-pki-types = { package = "rustls-pki-types", version = "1.9", features = ["std"] }
+pki-types = { package = "rustls-pki-types", version = "1.10", features = ["std"] }
```

Comment:
Since bogo and examples depend on rustls already (and aren't published) I don't think making this change for them is useful (though it can't hurt, either).

Reply:
(Arguably they should use the reexport?)

Reply:
https://github.com/rustls/rustls/pull/2158.